### ISSUE TITLE
[Feature] Create UI for a "detect bugs with similar domain names" feature

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#307]
When a user is entering a bug it might be useful to detect if bugs already exist with the same domain name.
1. It can help the user to find out that the bug has been already created.
2. It can help us to add automatically in the body a `seeAlso:` field with the list of bug numbers. When it's time to contact, we already know that there were previous bugs on the same topic. 

```
seeAlso: #123 #456
```

**Comment by miketaylr** [comment]
This depends on creating a database with URLs (or maybe not). But this would be good.

**Comment by hallvors** [comment]
It could use http://arewecompatibleyet.com/data/masterbugtable.js instead of a database..

**Comment by hallvors** [comment]
@miketaylr can we run cron jobs on webcompat.com server?

**Comment by miketaylr** [comment]
@hallvors yeah, I can set you up with a webcompat user login and you can knock yourself out. :bomb:

**Comment by hallvors** [comment]
This is a really important issue to tackle when we start getting more people sent over from input.mozilla and other places.

This issue needs some UI/UX design, namely: what exactly should we do when / if we discover that there are one or more issues already for a given domain name.  My suggestion would be to give the user a really easy way to transform their input to a comment on the existing issue, but this needs some clever UI.

Also, what if the existing issues are closed?

What if the domain is the same but the category of the existing issue is different?

Getting some design ideas on the table would be a great first step here.

**Comment by karlcow** [comment]
This issue has been split into two coding parts:

* Client-side implementation of "detect bugs with similar domain names" #864
* Backend implementation of "detect bugs with similar domain names" #865

Let me rephrase the title of this issue.

**Comment by miketaylr** [comment]
Assigning to @zoepage, she's going to work on this in Q2. 

I'd like to close #864 and #865 in the meantime -- we should revisit them for ideas and inspiration, but let's approach this with a fresh set of eyes (i.e., avoid some of the technical baggage in those issues).

**Comment by miketaylr** [comment]
Let's do this in Q3, post #1480.

**Comment by zoepage** [comment]
closing for now as we have #1826 as follow up.
